### ISSUE TITLE
gtk-doc: wrapper leaks into config

### ISSUE THREAD

**Issue by ebfe** [voidlinux/void-packages#7977]
```
/usr/share/gtk-doc/python/gtkdoc/config.py:
[...]
pkg_config = '/builddir/.xbps-gtk-doc/wrappers/arm-linux-gnueabihf-pkg-config'  
[...]
```

this breaks among others the mate-utils build #7969

**Comment by ebfe** [comment]
0d5058172e57a4b8cb9822f7942b9a204d78c069, fixes enough of it to get mate-utils/bamf to build. Still needs a proper fix.

**Comment by Gottox** [comment]
thanks!

I lost my notebook at the weekend. Cannot work on void till I get it back/get a replacement.

**Comment by Gottox** [comment]
Is this fixed in 0d5058172e57a4b8cb9822f7942b9a204d78c069?

**Comment by ebfe** [comment]
No, as written above that's just a hack just get a few packages to build that kept people from upgrading.
